drivers/ninaw10/nina_wifi_bsp.c

```diff
@@ -103,7 +114,11 @@ int nina_bsp_atomic_exit(void) {
 }
 
 int nina_bsp_read_irq(void) {
```

Comment:
I don't think this function is ever used, so maybe just wrap the whole thing in `#ifdef MICROPY_HW_NINA_GPIO0`.